etc/di.xml

```diff
@@ -53,4 +53,21 @@
                 <argument name="authSession" xsi:type="object">Magento\Backend\Model\Auth\Session\Proxy</argument>
         </arguments>
     </type>
+    <virtualType name="Fastly\Cdn\Model\RateLimit\Logger\Info" type="Magento\Framework\Logger\Handler\Base">
+        <arguments>
+            <argument name="fileName" xsi:type="string">/var/log/fastly-rate-limit.log</argument>
```

Comment:
This should not be a fixed path into /var/log since web user may not have write privileges. It should put the rate-limit log with the rest of the Magento log files.

Reply:
Yes, that path is not absolute to the system, but absolute to Magento's root. 
So, we are indeed putting the log with all other Magento logs 😄

Reply:
FWIW here is the devdocs reference: https://devdocs.magento.com/guides/v2.3/config-guide/log/custom-logger-handler.html